gradio/analytics.py

```diff
@@ -4,7 +4,6 @@
 import asyncio
 import json
 import os
-import pkgutil
```

Comment:
unrelated but not sure why we're just catching this now!

Reply:
I think its https://github.com/gradio-app/gradio/pull/5514/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711